bin/patches/omptests.patch

```diff
@@ -37,15 +37,36 @@ index 941704e..0846f05 100644
  results_folder := $(main_folder)/results
  error_log := $(results_folder)/error.log
  compile_log := $(results_folder)/compile.log
+diff --git a/Makefile.rules b/Makefile.rules
+index 5a37204..525a9d2 100644
+--- a/Makefile.rules
++++ b/Makefile.rules
+@@ -83,7 +83,8 @@ run: prepare_folder_to_run
+ 	@export LIBRARY_PATH
+ 	@export LD_LIBRARY_PATH
+ 	@echo "$(cmd) -o $(results_folder)/a.out" >> $(compile_cmd)
+-	@$(results_folder)/a.out 1>$(results_folder)/stdout 2>$(results_folder)/stderr ; true
++	@echo "HSA_XNACK=$(HSA_XNACK)"
++	@HSA_XNACK=$(HSA_XNACK) $(results_folder)/a.out 1>$(results_folder)/stdout 2>$(results_folder)/stderr ; true
+ 	@diff $(results_folder)/stdout expected > /dev/null || \
+ 	(if [ $$? ] ; then \
+ 		echo "#### Error #### Result mismatch for $(main_folder)" | tee -a $(error_log); \
+@@ -107,4 +108,4 @@ prepare_folder_to_run:
+ 	@echo "export LIBRARY_PATH=\"$(LIBRARY_PATH)\"" >  $(compile_cmd)
+ 	@echo "export LD_LIBRARY_PATH=\"$(LD_LIBRARY_PATH)\"" >>  $(compile_cmd)
+ 	@echo "cd $(main_folder)" >>  $(compile_cmd)
+-  
+\ No newline at end of file
++
 diff --git a/t-unified-barrier/Makefile b/t-unified-barrier/Makefile
 index cf913e4..64c8957 100644
 --- a/t-unified-barrier/Makefile
 +++ b/t-unified-barrier/Makefile
 @@ -2,6 +2,8 @@ include ../Makefile.defs
-
+ 
  EXTRAFLAGS := -O3 -Xclang -disable-llvm-optzns -fopenmp-version=45
-
-+export HSA_XNACK=${HSA_XNACK:-1}
+ 
++export HSA_XNACK=1
```

Comment:
these need to be
export HSA_XNACK?=1

so that if someone says export HSA_XNACK=0it is honored